DataFormats/DTRecHit/interface/DTDriftTimeParameters.icc

```diff
@@ -474,6 +475,9 @@ inline void THIS_CLASS::MB_DT_Get_grid_values(
     }
   }
 
+  // To silence LLVM analyzer warning
```

Comment:
I will check with DT experts

Reply:
This code, which is from ~20 years ago but still used in DT simulation, was designed (in a somewhat obsolete way) with the idea that the calling code is responsible to pass legal parameters only - which apparently is the case since it never crashed in the lifetime of CMS MC productions. 
Adding an additional assert before L481for iValue>=0 would certainly not harm.